### DISCUSSION TITLE
Feature Request: A Vuepress-next plugin

### DISCUSSION THREAD

**Discussion by authcompanion** [umami-software/umami#804] [Ideas]
Huge fan of umami - fits my use case just fine and many thanks to the contributors. 

**Feature request:** to have a [vuepress v2](https://github.com/vuepress/vuepress-next) plugin - similar approach to the [google analytics plugin](https://v2.vuepress.vuejs.org/reference/plugin/google-analytics.html).

Unfortunately, simply adding the tracking code via the [site config - head](https://v2.vuepress.vuejs.org/reference/config.html#head) and visiting a page results in multiple views being recorded. The views reported are increasing in number each time you navigate to a different site page (a browser refresh resets the views reported) - mirroring: https://github.com/mikecao/umami/issues/676 and related to https://github.com/mikecao/umami/issues/542 and https://github.com/mikecao/umami/issues/471

**Comment by mikecao** [comment]
Someone with Vue experience will have to tackle this one. I don't know anything about Vue.

**Comment by azat-io** [comment]
You can use https://github.com/azat-io/vuepress-plugin-umami-analytics
